packages/sdk/utils/src/sign-typed-data-utils.ts

```diff
@@ -59,9 +62,12 @@ export const EIP712_BUILTIN_TYPES = EIP712_ATOMIC_TYPES.concat(EIP712_DYNAMIC_TY
 // Regular expression used to identify and parse EIP-712 array type strings.
 const EIP712_ARRAY_REGEXP = /^(?<memberType>[\w<>\[\]_\-]+)(\[(?<fixedLength>\d+)?\])$/
 
-// Regular experssion used to identity EIP-712 integer types (e.g. int256, uint256, uint8).
+// Regular expression used to identity EIP-712 integer types (e.g. int256, uint256, uint8).
```

Comment:
optional: if you don't mind while you're at it, `identity` -> `identify`

Reply:
Oh yeah, missed that 👍